packages/react-components/src/components/ChatMessage/ChatMessageComponentAsMessageBubble.tsx

```diff
@@ -235,7 +240,9 @@ const MessageBubble = (props: ChatMessageComponentAsMessageBubbleProps): JSX.Ele
             // In doing so here we set the target of the flyout to be the message and
             // not the 3-dot menu button to position the flyout correctly.
             setChatMessageActionFlyoutTarget(messageRef);
-            props.onActionButtonClick(message, setMessageReadBy);
+            if (message.messageType === 'chat') {
```

Comment:
What would happen if user click 3-dot menu for blocked message?

Reply:
Good call out, added the check to disable menu if it's local user BlockedMessage.